src/coins.h

```diff
@@ -136,7 +136,10 @@ struct CCoinsCacheEntry
     CCoinsCacheEntry() noexcept = default;
     explicit CCoinsCacheEntry(Coin&& coin_) noexcept : coin(std::move(coin_)) {}
 
-    inline void AddFlags(uint8_t flags) noexcept { m_flags |= flags; }
+    //! Adding a flag also requires a self reference to the pair that contains
+    //! this entry in the CCoinsCache map and a reference to the head of the
+    //! flagged pair linked list.
+    inline void AddFlags(uint8_t flags, CoinsCachePair& self, CCoinsCacheEntry& head) noexcept { m_flags = flags; }
```

Comment:
hmmm, why did this become `=` instead of `|=`?

Reply:
Whoops bad rebase, thanks for catching!